### ISSUE TITLE
can i add with bootstrap 4?

### ISSUE THREAD

**Issue by amin007** [erusev/parsedown#734]

**Comment by Linnzh** [comment]
Maybe you can create a branch to design for bootstrap4?

**Comment by amin007** [comment]
@Linnzh , make a new branch?

**Comment by Linnzh** [comment]
@amin007 , I just make a suggestion. You can add a Bootstrap4 branch and add the corresponding class name to each tag in the branch code, or add a template to the generated HTML, etc. Of course, you can also fork the repository and customize it yourself (I tried Add the class name to the tag, but unfortunately this PR was not passed).
Of course, this is based on the fact that the class name is known and fixed. If you want to add freely, I think it will be very troublesome and lose the simplicity of this library.

**Comment by amin007** [comment]
@Linnzh , i have try add style in this library.

but if i add style, this library lose the simplicity

**Comment by Linnzh** [comment]
@amin007 , Yes. So what I did later, after generating the HTML snippet, use a <div> tag to wrap it, and then customize the style on this <div> tag. This will be simpler and can be changed at will.
